examples/http3-client.rs

```diff
@@ -241,6 +245,15 @@ fn main() {
                 quiche::h3::Header::new("user-agent", "quiche"),
             ];
 
+            // Additional headers
+            for header in &req_headers {
+                let header_split: Vec<&str> = header.splitn(2, ": ").collect();
+                // Convert header field to lowercase
+                let (field, value) =
+                    (&header_split[0].to_lowercase().to_string(), header_split[1]);
```

Comment:
This is already done when calling `Header::new()` so you don't need to do it explciitly here:
https://github.com/cloudflare/quiche/blob/master/src/h3/mod.rs#L464

Reply:
Updated. Thanks